torch/_inductor/codecache.py

```diff
@@ -1449,7 +1448,9 @@ def load_with_key(
 
             if (time_saved_ns := compiled_graph._time_taken_ns) is not None:
                 cache_info["time_saved_ns"] = time_saved_ns
-                add_remote_cache_time_saved(time_saved_ns, is_backward)
+                get_metrics_context().increment(
+                    "distributed_ephemeral_timeout_us", time_saved_ns // 1000
+                )
```

Comment:
note for later: actually `distributed_ephemeral_timeout_us` here is not accurate, the true time saved is `ephemeral_increase` lol